zipline/pipeline/factors/basic.py

```diff
@@ -54,6 +54,34 @@ def compute(self, today, assets, out, close):
         out[:] = (close[-1] - close[0]) / close[0]
 
 
+class PercentChange(SingleInputMixin, CustomFactor):
+    """
+    Calculates the percent change over the given window_length.
+
+    **Default Inputs:** None
+
+    **Default Window Length:** None
+
+    Notes
+    -----
+    Percent change is calculated as (new - old) / abs(old).
```

Comment:
```suggestion
    Percent change is calculated as ``(new - old) / abs(old)``.
```

This will make the formula render in monospace in our generated docs.